resources/views/checkout/steps/agreements.blade.php

```diff
@@ -0,0 +1,41 @@
+@php($checkoutAgreements ??= \Rapidez\Core\Models\CheckoutAgreement::all())
+@if (!$checkoutAgreements->count())
+    @return
+@endif
```

Comment:
We can change this with https://github.com/rapidez/blade-directives/pull/17 and bump the version to 0.7.0